atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/http/TimelockUtils.java

```diff
@@ -43,21 +46,26 @@ private TimelockUtils() {
     }
 
     private static List<String> hostnamesToEndpointUris(List<String> hosts) {
-        return Lists.transform(hosts, host -> String.format("https://%s:%d/%s", host, PORT, NAMESPACE));
+        return hosts.stream().map(host -> String.format("https://%s:%d", host, PORT)).collect(Collectors.toList());
     }
 
     private static <T> T createFromUris(MetricsManager metricsManager, List<String> endpointUris, Class<T> type) {
+        ServerListConfig serverListConfig = ImmutableServerListConfig.builder()
+                .addAllServers(endpointUris)
+                .sslConfiguration(SSL_CONFIGURATION)
+                .build();
+        // lock technically blocking, but OK to run with short timeout given the way the tests work.
+        AuxiliaryRemotingParameters build = AuxiliaryRemotingParameters.builder()
+                .shouldRetry(false)
+                .shouldLimitPayload(false)
+                .shouldSupportBlockingOperations(false)
+                .userAgent(UserAgent.of(UserAgent.Agent.of("atlasdb-jepsen", "1.2.3")))
```

Comment:
Why don't we use `UserAgent.Agent.DEFAULT_VERSION` anymore?

Reply:
I imagine it's slightly more identifiable, but can't think of another reason other than that 🤷‍♂

Reply:
Ah, that's fair, I forgot that existed - changed to that.